protobuf/api/service.proto

```diff
@@ -33,33 +33,50 @@ service Service {
 // The request's data for the `Create` API.
 message CreateServiceRequest {
   // Service's sid.
-  string sid = 1;
+  string sid = 1 [
+    (gogoproto.moretags) = 'validate:"required,printascii,max=63,domain"'
```

Comment:
sid is not required in the create request.
if it doesn't exist it is set by the service sdk.
```suggestion
    (gogoproto.moretags) = 'validate:"printascii,max=63,domain"'
```